docs/expand_markdown.py

```diff
@@ -58,7 +58,7 @@ def expand_markdown(
     input_markdown_path: Path = typer.Argument(...),
     output_markdown_path: Path = typer.Argument(...),
 ):
-    with open(input_markdown_path, "r") as input_file, open(
+    with open(input_markdown_path) as input_file, open(
```

Comment:
If `input_markdown_path` is `Path` type, please, use `input_markdown_path.open()` instead